l10n/de_DE.js

```diff
@@ -58,6 +58,14 @@ OC.L10N.register(
     "Redirect warning" : "Umleitungswarnung",
     "The previous page is sending you to %s" : "Die vorangegangene Seite leitet Sie zu %s weiter",
     "If you do not want to visit that page, you can return to <a href=\"%s\">the mail app</a>." : "Wenn Sie diese Seite nicht besuchen möchten, können Sie <a href=\"%s\">zur Mail-App</a> zurück kehren.",
-    "Continue to %s" : "Fortsetzen auf %s"
+    "Continue to %s" : "Fortsetzen auf %s",
+    "New message" : "Neue Nachricht",
+    "to" : "An",
+    "Subject" : "Betreff",
+    "Message …" : "Nachricht ...",
```

Comment:
`...` (3 dots) should be replaced with the Unicode [`HORIZONTAL ELLIPSIS`](http://www.fileformat.info/info/unicode/char/2026/index.htm) symbol (as in the original non-translated version).